system/handlers/admin/DataManager.cfc

```diff
@@ -344,6 +344,76 @@
 			);
 		</cfscript>
 	</cffunction>
+	<cffunction name="batchEditField" access="public" returntype="void" output="false">
+		<cfargument name="event"             type="any"     required="true" />
+		<cfargument name="rc"                type="struct"  required="true" />
+		<cfargument name="prc"               type="struct"  required="true" />
+		<cfscript>
+			var objectName   = rc.object;
```

Comment:
A number of issues with the code block below:

* Variables with no `var` scope (`fieldAttributes`, `fieldType`, `formControl`).
* `formControl` - let's look for a better solution here, ideally we should have something like: `renderFormControlForObjectField()`; I can help out here
* `manyToManySelect` special case: we really shouldn't need this. Need to understand what the problem is in order to find the right fix
* Breadcrumb title: need a title especially for "Batch edit". So: `uri="cms:datamanager.batchedit.breadcrumb.title", data=[ objectTitle, fieldTitle ]` (where object title and field title are translated labels for the object and field